ClassTest.py

```diff
@@ -70,6 +70,17 @@ def testRenameNonExistentClass(self):
         collection = ClassCollection()
         collection.addClass("A")
         self.assertRaises(KeyError, collection.renameClass, "C", "D")
-   
+
+    def testDeleteClassWithRelationshipandField(self):
```

Comment:
I would probably put some sort of assert in this test, like an assertNotIn to see if A and B are no longer in the classDict.